assets/common/environments/python-sdk-v2/context/conda_dependencies.yaml

```diff
@@ -10,3 +10,5 @@ dependencies:
     - azure-identity=={{latest-pypi-version}}
     - azureml-core=={{latest-pypi-version}}
     - applicationinsights=={{latest-pypi-version}}
```

Comment:
could you validate with removing applicationinsights and azure-core?

Reply:
Sure